activerecord/lib/active_record/migration.rb

```diff
@@ -106,17 +106,16 @@ def initialize
   #
   # == Available transformations
   #
+  # === Creation
+  #
+  # * <tt>create_join_table(table_1, table_2, options)</tt>: Creates a new join
+  #   table with the name created using the lexical order of the first two
```

Comment:
How about `table having its name as the lexical (...)`?
Should we also note that the table object will be provided to a block, just like `create_table`?